lib/engine/game/base.rb

```diff
@@ -905,6 +905,20 @@ def add_extra_tile(tile)
         extra_cities.each { |c| @_cities[c.id] = c }
       end
 
+      def share_ability_money_on_par(corporation)
+        return unless corporation.capitalization == :incremental
+
+        @companies.each do |company|
+          next unless (ability = company.abilities(:shares))
+          next unless corporation.name == ability.shares.first.corporation.name
+
+          amount = ability.shares.sum { |share| corporation.par_price.price * share.num_shares }
+          @bank.spend(amount, corporation)
+          @log << "#{corporation.name} receives #{format_currency(amount)}
+                   capital from shares provided by #{company.name}"
```

Comment:
do we use the term captial anywhere else? it's usually just

x recievy $1 from z ?

Reply:
That's funny as it was what I had originally. I'll switch it back.